modules/patch-argocd-instance.adoc

```diff
@@ -0,0 +1,20 @@
+// Module is included in the following assemblies:
+//
+// * configuring-resource-quota
+
+:_content-type: PROCEDURE
+[id="patch-argocd-instance_{context}"]
+= Patching Argo CD instance to update the resource requirements
+
+[role="_abstract"]
+You can update the resource requirements for all or any of the workloads post installation. 
+
+Update the `Application Controller` resource requests of an Argo CD instance in the Argo CD namespace.
```

Comment:
I think it is standard practice to add a title before the procedure:

.Procedure
Update the `Application Controller` resource requests...

If you don't want it to show up in the toc, add [discrete] before it